src/index.js

```diff
@@ -28,6 +28,7 @@ const getParams = () => {
     currentRepo,
     githubToken,
     personalToken,
+    team: JSON.parse(core.getInput('team')),
```

Comment:
@aleksanderkaminski nice! You want to get the stats for a specific team? That's a good idea. I might merge to `master`.

Reply:
Hey there! I am not entirely sure if you're being serious, this code was written literally in 20m, I am not sure if it's ready for production!

But indeed the idea that @jacekll had was to have statistics for the entire team instead of team members. Maybe if it will indeed bring value we can work on a proper solution. 🤔